src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -31,44 +31,72 @@
     private BookService bookService;
 
     @GetMapping("/greeting")
+    /**
+     * @param name the name for show in the view
+     * @param model the model to add attributes for show in the view
+     * @return the view to show the greet
+     */
     public String greeting(@RequestParam(name = "name", required = false, defaultValue = "World") String name,
             Model model) {
         model.addAttribute(NAME, name);
         return GREETING_VIEW;
     }
 
     @PostMapping
+    /**
+     * @param bookVO the book with the attributes that can choose the user to create it
```

Comment:
Wherever an object is declared on java doc, I recommend you to declare it like this: {@link BookVO}, so when you generate the documentation, you will see a link to this Object :) 

Replace it in every case.

Reply:
Good tip Mati, I pushed the change!